backend/config/corpora-api.yml

```diff
@@ -329,6 +329,10 @@ paths:
                 properties:
                   dataset_uuid:
                     $ref: "#/components/schemas/dataset_uuid"
+                  processing_status:
+                    type: string
+                    enum:
+                      [ PENDING, SUCCESS,FAILURE]
```

Comment:
Created a PR to fix this and add prettier to format `yaml`! https://github.com/chanzuckerberg/corpora-data-portal/pull/869

PTAL thank you :D